OpenRA.Mods.TS/UtilityCommands/LegacySequenceImporter.cs

```diff
@@ -0,0 +1,275 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2016 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using OpenRA.Mods.Common.FileFormats;
+
+namespace OpenRA.Mods.TS.UtilityCommands
+{
+	class ImportLegacySequenceCommand : IUtilityCommand
+	{
+		public bool ValidateArguments(string[] args)
+		{
+			return args.Length >= 2;
+		}
+
+		public string Name { get { return "--sequence-import"; } }
+
+		IniFile file;
+		MapGrid grid;
+
+		[Desc("FILENAME", "Convert ART.INI to the OpenRA sequence definition format.")]
+		public void Run(ModData modData, string[] args)
+		{
+			// HACK: The engine code assumes that Game.modData is set.
+			Game.ModData = modData;
+
+			grid = Game.ModData.Manifest.Get<MapGrid>();
+
+			file = new IniFile(File.Open(args[1], FileMode.Open));
+
+			foreach (var section in file.Sections)
+			{
+				var sequence = section.GetValue("Sequence", string.Empty);
+				if (!string.IsNullOrEmpty(sequence))
+					ImportInfantrySequences(section, sequence);
+
+				var foundation = section.GetValue("Foundation", string.Empty);
+				if (!string.IsNullOrEmpty(foundation))
+					ImportBuildingSequences(section);
+			}
+		}
+
+		WPos CenterOfIsometricCell(CPos cell)
+		{
+			return new WPos(512 * (cell.X - cell.Y + 1), 512 * (cell.X + cell.Y + 1), 0);
+		}
+
+		void ImportBuildingSequences(IniSection section)
+		{
+			Console.WriteLine(section.Name + ":");
+
+			Console.WriteLine("\tDefaults:");
+
+			var foundation = section.GetValue("Foundation", string.Empty);
+			if (!string.IsNullOrEmpty(foundation))
```

Comment:
Added. This won't hide as you commented on a line that isn't affected by your suggestions.